scripts/storage-facts.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+#this script generates storage (ZFS, iSCSI) facts out of the hostvars of each VM
```

Comment:
There are example configurations for every role in their respective repositories.
But I commented the script further.